.cursor/rules/20-solidity-scripts.mdc

```diff
@@ -0,0 +1,11 @@
+---
+name: Solidity deploy/update scripts
+description: Foundry deployment/update script patterns
+globs:
+  - 'script/**/*.s.sol'
+  - 'script/**/*.sol'
+---
+
+- Use `DeployScriptBase`/`UpdateScriptBase` patterns: `Deploy*.s.sol` uses `deploy(type(Contract).creationCode)`; only add `getConstructorArgs()` when needed and read from JSON via `stdJson`.
+- Update scripts call `update(\"{ContractName}\")` and override `getExcludes()` for selector skips.
```

Comment:
what about giving more context here for getExcludes() and add if selectors from other facet's can conflicts when calling diamond for example or functions that we want to call only on diamondCut?